config.yaml

```diff
@@ -15,7 +15,12 @@ options:
     type: boolean
     default: false
     description: |
-      Toggle installation from ubuntu archive vs the docker PPA
+      Toggle installation from ubuntu archive vs the docker PPA (DEPRECATED please use docker_runtime instead)
+  docker_runtime:
+    type: string
+    default: ""
```

Comment:
I am not a vary big fan of this "whatever" maps to "deploy from apt". I would prefer if we had two more options "apt" -> "use docker from default apt repo" and "auto" -> "do what you think is best". We might get into trouble during upgrades if we leave this free text option.